run_tests.sh

```diff
@@ -18,15 +10,19 @@ else
     DATABASE_UP=0
 fi
 
+docker ps | grep '\->8545' > /dev/null
```

Comment:
We should start everything together, and then sleep for both

`docker-compose up -d ganache db`